src/main/kotlin/pl/newbies/event/domain/service/EventService.kt

```diff
@@ -1,9 +1,11 @@
 package pl.newbies.event.domain.service
 
 import kotlinx.datetime.Clock
-import org.jetbrains.exposed.sql.SizedCollection
-import org.jetbrains.exposed.sql.and
+import org.jetbrains.exposed.sql.*
```

Comment:
whats this 🐜

Reply:
I'll ignore wildcard imports from now on and configure lint / editor later.